notes.txt

```diff
@@ -0,0 +1,8 @@
+This bot returns the top hit from google news related to the topic sent.
```

Comment:
I would suggest moving this line into the readme, instead of having it's own file